cms/djangoapps/contentstore/views/course.py

```diff
@@ -1207,7 +1207,7 @@ def settings_handler(request, course_key_string):  # lint-amnesty, pylint: disab
                 courses, in_process_course_actions = get_courses_accessible_to_user(request)
                 # exclude current course from the list of available courses
                 courses = (course for course in courses if course.id != course_key)
-                if courses:
+                if courses:  # pylint: disable=using-constant-test
```

Comment:
This was a good warning, and we should not silence it.  `courses` is a generator, which is always true.  Originally the code made a list, and this check was checking if it was empty, but you can't check if a generator is empty.

Notice also, the function is `_process_courses_list` even though it isn't getting a list anymore.